Client/mods/deathmatch/logic/CClientGame.cpp

```diff
@@ -2528,6 +2528,7 @@ void CClientGame::AddBuiltInEvents()
     m_Events.AddEvent("onClientPedStep", "foot", nullptr, false);
 
     // Vehicle events
+    m_Events.AddEvent("onClientVehicleCreate", "", NULL, false);
```

Comment:
The event should be called `onClientVehicleCreated`

Reply:
Done